tools/recalc.py

```diff
@@ -29,6 +30,7 @@
     from app.constants.gamemodes import GameMode
     from app.objects.beatmap import ensure_local_osu_file
     import app.settings
+    import app.state
```

Comment:
i think this should still work okay, just limits import scope a bit
```suggestion
    import app.state.service
```

Reply:
done